drivers/flash/CMakeLists.txt

```diff
@@ -75,8 +75,10 @@ endif()
 if(CONFIG_SOC_FLASH_STM32)
   if(CONFIG_SOC_SERIES_STM32H7X)
     zephyr_library_sources_ifdef(CONFIG_DT_HAS_ST_STM32H7_FLASH_CONTROLLER_ENABLED flash_stm32h7x.c)
+    zephyr_library_sources_ifdef(CONFIG_FLASH_EX_OP_ENABLED flash_stm32_ex_op.c)
```

Comment:
not sure, the 
`(CONFIG_SOC_SERIES_STM32WBX)`
have not

Reply:
`CONFIG_FLASH_EX_OP_ENABLED` still needs to be enabled manually and `CONFIG_FLASH_HAS_EX_OP` must be selected, so I think it can be safely moved under `CONFIG_SOC_FLASH_STM32`.